ff/src/fields/models/fp/montgomery_backend.rs

```diff
@@ -5,6 +5,11 @@ use crate::{biginteger::arithmetic as fa, BigInt, BigInteger};
 
 /// A trait that specifies the constants and arithmetic procedures
 /// for Montgomery arithmetic over the prime field defined by `MODULUS`.
+///
+/// # Note
+/// Manual implementation of this trait is not recommended unless one wishes
+/// to special arithmetic methods. Instead, the [`MontConfig`][`ark_ff_macros::MontConfig`]
```

Comment:
"wish to use special..." or "wish to specify"?

Reply:
Oops, good catch.

```suggestion
/// to specialize arithmetic methods. Instead, the [`MontConfig`][`ark_ff_macros::MontConfig`]
```